### ISSUE TITLE
Wrong PHP_VERSION check

### ISSUE THREAD

**Issue by ktrace** [torrentpier/torrentpier#224]
library/includes/init_bb.php:
```
 if (PHP_VERSION < '7.0.8') {
    die('TorrentPier requires PHP version 7.0.8 and above (used code and packages requirement). Your PHP version is ' . PHP_VERSION);
}
```
TorrentPier requires PHP version 7.0.8 and above (used code and packages requirement). Your PHP version is 7.0.15

**Comment by VasyOk** [comment]
#225 Fixed for `develop`

**Comment by ktrace** [comment]
```
-if (version_compare(PHP_VERSION, '7.0.8', '>')) {
+if (version_compare(PHP_VERSION, '7.0.8', '<')) {
```

**Comment by ktrace** [comment]
Plz fix, you fail with version > 7.0.8

**Comment by Exile37** [comment]
@ktrace fixed in https://github.com/torrentpier/torrentpier/commit/8a98517b55d1d8fd49bc528b27e22d3bd1d2c1ce, thank you 👍
